pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "pytroll-collectors"
+dynamic = ["version"]
+description = "Pytroll data collectors"
+readme = "README.md"
```

Comment:
Does pyproject.toml README definition handle the markdown versus restructuredtext determination for you? In `setup.py` you need to define `long_description_content_type`.